recipe/meta.yaml

```diff
@@ -19,13 +19,12 @@ requirements:
   host:
     - pip
     - python
-    - setuptools
-    - wheel
+    - hatchling
   run:
     - python
     - importlib-metadata  # [py<38]
     - snowflake-connector-python <4.0.0
-    - sqlalchemy <2.0.0,>=1.4.0
+    - sqlalchemy >=1.4.19
 
 test:
```

Comment:
Is it possible to run some of the tests under `tests/` ?

Reply:
For `doc_url`, would this be a better alternative : https://docs.snowflake.com/en/developer-guide/python-connector/sqlalchemy ?

Reply:
I've updated doc_url

Reply:
Tests require connecting to snowflake, I don't think we should do that.

Reply:
If all the tests require connecting to snowflake, I agree we should skip them